gulpfile.babel.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
I don't care for `"use strict"`, to have it here and nowhere else is bothersome.